wordpress_rock/rockcraft.yaml

```diff
@@ -0,0 +1,311 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: wordpress
+summary: Wordpress rock
+description: Wordpress OCI image for the Wordpress charm
+base: ubuntu:20.04
```

Comment:
I wonder if we should take this opportunity (if possible) to build based on 22.04

Reply:
I think it's mentioned somewhere that launchpad integration plugin for wordpress does not work with PHP8 (shipped as default when installing php on 22.04) So until that is resolved I think we still need to stay on PHP7